2022-03-09 15:45:34,437 INFO    MainThread:169972 [wandb_setup.py:_flush():71] setting env: {'api_key': '***REDACTED***'}
2022-03-09 15:45:34,437 INFO    MainThread:169972 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-03-09 15:45:34,437 INFO    MainThread:169972 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-03-09 15:45:34,438 INFO    MainThread:169972 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-09 15:45:34,438 INFO    MainThread:169972 [wandb_init.py:_log_setup():371] Logging user logs to /home/haiduong/Documents/Project 3/TorchSemiSeg/CPS_Kaggle/wandb/run-20220309_154534-2exn33eq/logs/debug.log
2022-03-09 15:45:34,438 INFO    MainThread:169972 [wandb_init.py:_log_setup():372] Logging internal logs to /home/haiduong/Documents/Project 3/TorchSemiSeg/CPS_Kaggle/wandb/run-20220309_154534-2exn33eq/logs/debug-internal.log
2022-03-09 15:45:34,439 INFO    MainThread:169972 [wandb_init.py:init():404] calling init triggers
2022-03-09 15:45:34,439 INFO    MainThread:169972 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-03-09 15:45:34,439 INFO    MainThread:169972 [wandb_init.py:init():460] starting backend
2022-03-09 15:45:34,439 INFO    MainThread:169972 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-09 15:45:34,457 INFO    MainThread:169972 [backend.py:ensure_launched():216] starting backend process...
2022-03-09 15:45:34,464 INFO    MainThread:169972 [backend.py:ensure_launched():221] started backend process with pid: 170023
2022-03-09 15:45:34,466 INFO    MainThread:169972 [wandb_init.py:init():469] backend started and connected
2022-03-09 15:45:34,469 INFO    MainThread:169972 [wandb_init.py:init():533] updated telemetry
2022-03-09 15:45:34,481 INFO    MainThread:169972 [wandb_init.py:init():563] communicating current version
2022-03-09 15:45:35,406 INFO    MainThread:169972 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-09 15:45:35,406 INFO    MainThread:169972 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-09 15:45:35,829 INFO    MainThread:169972 [wandb_init.py:init():606] starting run threads in backend
2022-03-09 15:45:40,844 INFO    MainThread:169972 [wandb_run.py:_console_start():1810] atexit reg
2022-03-09 15:45:40,845 INFO    MainThread:169972 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-09 15:45:40,846 INFO    MainThread:169972 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-09 15:45:40,851 INFO    MainThread:169972 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-09 15:45:40,852 INFO    MainThread:169972 [wandb_init.py:init():633] run started, returning control to user process
2022-03-09 15:45:54,876 INFO    MainThread:169972 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-09 15:45:54,879 INFO    MainThread:169972 [wandb_run.py:_restore():1752] restore
2022-03-09 15:46:02,383 INFO    MainThread:169972 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-03-09 15:46:07,485 INFO    MainThread:169972 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-03-09 15:46:12,586 INFO    MainThread:169972 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-03-09 15:46:17,687 INFO    MainThread:169972 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
2022-03-09 15:46:22,788 INFO    MainThread:169972 [wandb_run.py:_wait_for_finish():1912] got exit ret: None
